docs/application/dotnet/samples/tizen-samples.md

```diff
@@ -254,6 +251,7 @@
 			</td>
 			<td>
 			<p><strong>FirstScreen</strong></p>
+                        <p>FirstScreen is a sample application that demonstrates how to launch the apps on the TV.</p>
```

Comment:
Review result from annie-samsung
```
If FirstScreen sample app lets you to launch only one app on the TV, then please change the sentence to:

FirstScreen is a sample application that demonstrates how to launch an app on the TV.
```

Or, if current sentence is correct, please let me know.

Reply:
Not for only one app, FirstScreen can launch several apps

Reply:
Thanks for the check.